test/integration/integration_test.go

```diff
@@ -66,5 +66,6 @@ func TestIntegration(t *testing.T) {
 		t.Run("TestCrud", TestCrud)
 		t.Run("TestReplicaSchedulingPreference", TestReplicaSchedulingPreference)
 		t.Run("TestMultiClusterServiceDNS", TestMultiClusterServiceDNS)
+		t.Run("TestJobSchedulingPreference", TestJobSchedulingPreference)
```

Comment:
Consider rewriting as an e2e test.

Integration tests can only run against managed fixture, but e2e tests can target both managed and unmanaged (i.e. deployed clusters) fixture.  Only tests that can run with unmanaged fixture can validate interaction with the kubernetes control plane (in this case the job controller).

Reply:
Agree, I have reduced the integration tests to a single test to validate the path; would be submitting another PR for e2e tests.